### PR TITLE
system_config.rb: show system Ruby info on Linux

### PR DESCRIPTION
**Pull request by maxim-belkin**
I propose to report the version of system Ruby in `config` as this info might be helpful for debugging purposes.

-----
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
This should be Linux only (i.e. in `os/extend`) as we're not gonna use this on macOS.

File: Library/Homebrew/system_config.rb [hunk 1]
```diff
@@ -74,6 +74,13 @@ def describe_homebrew_ruby_version
       end
     end
 
+    def system_ruby_version
+      out, _, status = system_command("/usr/bin/ruby", args: ["-e", "p RUBY_VERSION"], print_stderr: false)
```
Comment (MikeMcQuaid) [inline Library/Homebrew/system_config.rb:78]:
```suggestion
      out, _, status = system_command("/usr/bin/ruby", args: ["-e", "puts RUBY_VERSION"], print_stderr: false)
```

This should remove the need for the `.chomp`

File: Library/Homebrew/system_config.rb [hunk 2]
```diff
@@ -156,6 +163,7 @@ def dump_verbose_config(f = $stdout)
 
       f.puts hardware if hardware
       f.puts "Homebrew Ruby: #{describe_homebrew_ruby}"
+      f.puts "System Ruby: #{system_ruby_version}" unless RUBY_PATH == "/usr/bin/ruby"
```
Comment (MikeMcQuaid) [inline Library/Homebrew/system_config.rb:166]:
```suggestion
      f.puts "System Ruby: #{system_ruby_version}" if RUBY_PATH != SYSTEM_RUBY_PATH
```

and define `SYSTEM_RUBY_PATH` above.

Comment (maxim-belkin) [reply Library/Homebrew/system_config.rb]: > 

> and define `SYSTEM_RUBY_PATH` above

will do

**Review by dawidd6 — APPROVED** [review]

File: Library/Homebrew/extend/os/linux/system_config.rb [hunk 3]
```diff
@@ -27,12 +28,20 @@ def formula_linked_version(formula)
       "N/A"
     end
 
+    def system_ruby_version
```
Comment (MikeMcQuaid) [inline Library/Homebrew/extend/os/linux/system_config.rb:31]:
```suggestion
    def host_ruby_version
```

File: Library/Homebrew/extend/os/linux/system_config.rb [hunk 4]
```diff
@@ -4,6 +4,7 @@
 require "os/linux/glibc"
 
 module SystemConfig
+  SYSTEM_RUBY_PATH = "/usr/bin/ruby"
```
Comment (MikeMcQuaid) [inline Library/Homebrew/extend/os/linux/system_config.rb:7]:
```suggestion
  HOST_RUBY_PATH = "/usr/bin/ruby"

```

File: Library/Homebrew/extend/os/linux/system_config.rb [hunk 5]
```diff
@@ -27,12 +28,20 @@ def formula_linked_version(formula)
...
+      out
+    end
+
     def dump_verbose_config(out = $stdout)
       dump_generic_verbose_config(out)
       out.puts "Kernel: #{`uname -mors`.chomp}"
       out.puts "OS: #{OS::Linux.os_version}"
       out.puts "Host glibc: #{host_glibc_version}"
       out.puts "/usr/bin/gcc: #{host_gcc_version}"
+      out.puts "/usr/bin/ruby: #{system_ruby_version}" if RUBY_PATH != SYSTEM_RUBY_PATH
```
Comment (MikeMcQuaid) [inline Library/Homebrew/extend/os/linux/system_config.rb:44]:
```suggestion
      out.puts "/usr/bin/ruby: #{host_ruby_version}" if RUBY_PATH != HOST_RUBY_PATH
```

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for this @maxim-belkin!

**Pull request merged**